tests/unit/components/sl-calendar-month-test.js

```diff
@@ -0,0 +1,62 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+
+moduleForComponent( 'sl-calendar-month', 'Unit - component: sl-calendar-month' );
+
+test( 'Default state is inactive', function( assert ) {
+    var component  = this.subject(),
+        $component = this.render();
+
+    assert.strictEqual( component.get( 'active' ), false, 'Component is not active' );
+    assert.strictEqual( $component.hasClass( 'active' ), false, 'Component does not have "active" class' );
+});
+
+test( 'Component has "month" class by default', function( assert ) {
+    var $component;
+
+    this.subject();
+    $component = this.render();
+
+    assert.ok( $component.hasClass( 'month' ), '"month" class is present' );
+});
+
+test( 'Active state is set correctly', function( assert ) {
+    var component  = this.subject({ active: true }),
+        $component = this.render();
+
+    assert.ok( component.get( 'active' ), 'Component is active' );
+    assert.ok( $component.hasClass( 'active' ), '"active" class is present' );
+});
+
+test( 'Action binding sends action', function( assert ) {
```

Comment:
Should also test that the expected value (in this case "month") is passed along.